### PR TITLE
Display SWC/DC instructor status in names (patched)

### LINKED ISSUE #759: Always display badges with people's names
Always display icons for SWC/DC instructor badges alongside people's names, so that (for example) when we're viewing the list of people who participated in an instructor training workshop, we can see which ones are now qualified and now.

From a conversation with @tracykteal

### PR DESCRIPTION
**Pull request by chrismedrela**
This is a patch created by @gvwilson (#762) with my minor tweaks.

Closes #759.

- Modifying display of all persons to distinguish Software Carpentry and Data Carpentry instructors.
- Modifying display of tasks associated with Event to distinguish Software Carpentry and Data Carpentry instructors.

### CODE REVIEW

File: workshops/views.py [hunk 1]
```diff
@@ -435,18 +435,14 @@ def all_persons(request):
 
     filter = PersonFilter(
         request.GET,
-        queryset=Person.objects.all().defer('notes')  # notes are too large
+        queryset=Person.objects.defer('notes') # notes are too large
+                       .annotate(is_swc_instructor=Sum(Case(When(badges__name='swc-instructor', then=1), default=0, output_field=IntegerField())),
+                                 is_dc_instructor=Sum(Case(When(badges__name='dc-instructor', then=1), default=0, output_field=IntegerField())))
```
Comment (pbanaszkiewicz) [inline workshops/views.py]: Lines definitely way too long (we try to stick to 79 columns).

File: workshops/views.py [hunk 2]
```diff
@@ -950,6 +946,8 @@ def event_details(request, event_ident):
 
     tasks = Task.objects.filter(event__id=event.id) \
                         .select_related('person', 'role') \
+                        .annotate(person_is_swc_instructor=Sum(Case(When(person__badges__name='swc-instructor', then=1), default=0, output_field=IntegerField())),
+                                  person_is_dc_instructor=Sum(Case(When(person__badges__name='dc-instructor', then=1), default=0, output_field=IntegerField()))) \
```
Comment (pbanaszkiewicz) [inline workshops/views.py]: Lines too long too.

**Comment by pbanaszkiewicz** [comment]
Alright, @chrismedrela fix the styling issues and I'll merge. Thanks!

**Comment by chrismedrela** [comment]
@pbanaszkiewicz, done!

File: workshops/views.py [hunk 3]
```diff
@@ -1589,6 +1606,8 @@ def instructors(request):
     instructor_badges = Badge.objects.instructor_badges()
     instructors = Person.objects.filter(badges__in=instructor_badges) \
                                 .filter(airport__isnull=False) \
+                                .annotate(is_swc_instructor=Sum(Case(When(badges__name='swc-instructor', then=1), default=0, output_field=IntegerField())),
+                                          is_dc_instructor=Sum(Case(When(badges__name='dc-instructor', then=1), default=0, output_field=IntegerField()))) \
```
Comment (pbanaszkiewicz) [inline workshops/views.py]: These are too long too, sorry I didn't mark them previously!

**Comment by chrismedrela** [comment]
@pbanaszkiewicz, done!

**Comment by pbanaszkiewicz** [comment]
Excellent, thank you @chrismedrela!

**Pull request merged**